tyk-pro/values.yaml

```diff
@@ -158,17 +159,19 @@ dash:
   org:
     name: Default Org
     # Set this value to the domain of your developer portal
-    cname: "localhost"
+    cname: tyk-portal.local
 
 portal:
+  enabled: true
```

Comment:
I would suggest that enabled should be false by default. Then provide clear instruction on how to enable it.

Optimise for simplicity, performance and security. Don't make users turn things off if they don't need it, make them explicitly enable things if they do need it. But provide sensible defaults for when they do turn it on.

Reply:
I think we actually dont need this at all? The only place it is used is in invoking the portal ingress chart. I would rather the portal exists and then whether to expose it outside the cluster is the users choice at that point.

Reply:
I agree with you @joshblakeley only reason I added this is consistency. Is that something we want to keep for that or no? Because otherwise it is not providing any value. @asoorm that sounds good to me as well if we do decide to keep it I will set it to false by default

Reply:
I also disagree it should be false if you did keep it.